app/models/label/LabelTable.scala

```diff
@@ -163,19 +169,24 @@ object LabelTable {
    * Date: Sep 2, 2016
    */
   def selectTopLabelsAndMetadataByUser(n: Int, userId: UUID): List[LabelMetadata] = db.withSession { implicit session =>
-    val selectQuery = Q.queryNA[(Int, Int, String, String, java.sql.Timestamp, String, Option[Int], Option[String])](
+    val selectQuery = Q.queryNA[(Int, Int, String, String, java.sql.Timestamp, String, Option[Int],
+                                  Boolean, Option[String])](
```

Comment:
I actually handled that in the query. The query will _always_ return a true or a false.